content/docs/user-guide/experiment-management/cleaning.md

```diff
@@ -0,0 +1,262 @@
+
+Supplying `--workspace` flag to `dvc exp gc` causes all experiments to be
+removed **except** those in the current workspace.
+
+```dvc
+$ dvc exp list --all
+main:
+   exp-aaa000
+   exp-aaa111
+   exp-aaa222
+other:
+   exp-bbb333
+   exp-bbb444
+another:
+   exp-ccc555
+   exp-ccc666
+   exp-ccc777
+```
+
+Issuing `dvc exp gc --workspace` removes experiments in `other` and `another`
+branches in this example.
+
+```dvc
+$ dvc exp gc --workspace
+$ dvc exp list --all
+main:
+   exp-abc000
+   exp-abc111
+   exp-abc222
+```
+
+### Keeping experiments in all branches
+
+DVC can create a branch for an experiment using `dvc exp branch` command.
+
+In cases where you want to clean up the experiments _except_ those in the
+branches, you can use `--all-branches` flag.
+
+```dvc
+$ dvc exp show --all-branches
+```
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
+┃ Experiment              ┃ Created      ┃    acc ┃ model.conv_units ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━━━━━┩
+│ workspace               │ -            │      - │ 64               │
+│ cnn-48                  │ 09:11 AM     │ 0.9131 │ 48               │
+│ main                    │ Jul 21, 2021 │ 0.9189 │ 16               │
+│ ├── dac711b [cnn-32]    │ 09:16 AM     │ 0.9152 │ 32               │
+│ ├── 7cd3ae7 [cnn-48]    │ 09:11 AM     │ 0.9131 │ 48               │
+│ ├── ab585b5 [cnn-24]    │ 09:06 AM     │ 0.9135 │ 24               │
+│ ├── 7d51b55 [exp-44136] │ 09:01 AM     │ 0.9151 │ 16               │
+│ └── 7feaa1c [exp-78ede] │ Aug 02, 2021 │ 0.9151 │ 16               │
+│ 8583124                 │ Jul 20, 2021 │ 0.9132 │ 17               │
+└─────────────────────────┴──────────────┴────────┴──────────────────┘
+```
+
+Supplying `--all-branches` keeps only the experiments in branch tips. Any
+experiment that's not promoted to a branch is removed this way.
+
+```dvc
+$ dvc exp gc --all-branches
+WARNING: This will remove all experiments except those derived from the workspace and all git branches of the current repo.
+Are you sure you want to proceed? [y/n] y
+Removed 6 experiments. To remove unused cache files use 'dvc gc'.
+```
+
+The resulting `dvc exp show` table is as the following:
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━━━━━┓
+┃ Experiment              ┃ Created      ┃    acc ┃ model.conv_units ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━━━━━┩
+│ workspace               │ -            │      - │ 64               │
+│ cnn-48                  │ 09:11 AM     │ 0.9131 │ 48               │
+│ main                    │ Jul 21, 2021 │ 0.9189 │ 16               │
+│ 8583124                 │ Jul 20, 2021 │ 0.9132 │ 17               │
+└─────────────────────────┴──────────────┴────────┴──────────────────┘
+```
+
+### Keeping experiments in all tags
+
+- [ ] `--all-tags` example
+
+### Keeping experiments in all commits
+
+- [ ] `--all-commits` example
```

Comment:
I'd start with this example since it's the least destructive. Then all branches and tags combined in a single section and finally `-w`

Reply:
If the user keeps all experiments in commits/branches, this is the least destructive. If they keep all in the workspace without bothering with the tags/commits, this is the most destructive.

Reply:
I prefer to order by _scope_, rather than destructivity.

Reply:
OK